### PR TITLE
New version: Microsoft.Teams version 1.6.00.20074

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115729&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  Scope: machine
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-x64/1.6.00.20074/Teams_windows_x64.exe
+  InstallerSha256: C6F0CB12B85377B15D27B3796127FE346292DC117F49BA24334B351A7F8830FB
+  ProductCode: '{731F6BAA-A986-45A4-8936-7C3AAAAA760B}'
+- Architecture: arm64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-arm64/1.6.00.20074/Teams_windows_arm64.exe
+  InstallerSha256: D1F90174DF2A6D83A979917CC731E964CD64C7D637C1C3AF43D28FA4672B2570
+  InstallerSwitches:
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:51]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    Silent: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:52]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    SilentWithProgress: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:53]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  Scope: machine
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.6.00.20074/Teams_windows.msi
+  InstallerSha256: C08AF6D2FFE440463435A00E247657FFA332E8B75F78F64BF20AFD7354507AF5
+  ProductCode: '{39AF0813-FA7B-4860-ADBE-93B9B214B914}'
+- Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows-x64/1.6.00.20074/Teams_windows_x64.exe
+  InstallerSha256: C6F0CB12B85377B15D27B3796127FE346292DC117F49BA24334B351A7F8830FB
+  InstallerSwitches:
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:35]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    Silent: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:36]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    SilentWithProgress: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:37]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+UpgradeBehavior: install
+ReleaseDate: 2023-07-20
+RequireExplicitUpgrade: true
+Installers:
+- Architecture: x86
+  InstallerType: msi
+  Scope: user
+  InstallerUrl: https://statics.teams.cdn.office.net/production-windows/1.6.00.20074/Teams_windows.msi
+  InstallerSha256: C08AF6D2FFE440463435A00E247657FFA332E8B75F78F64BF20AFD7354507AF5
+  InstallerSwitches:
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:18]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    Silent: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:19]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+    SilentWithProgress: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:20]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 10]
```diff
@@ -0,0 +1,62 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Teams
+PackageVersion: 1.6.00.20074
+InstallerSwitches:
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:6]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Teams
+PackageVersion: 1.6.00.20074
+InstallerSwitches:
+  Silent: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:7]:
```suggestion
```

File: manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml [hunk 12]
```diff
@@ -0,0 +1,62 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Teams
+PackageVersion: 1.6.00.20074
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
```
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/1.6.00.20074/Microsoft.Teams.installer.yaml:8]:
```suggestion
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
🤔

**Comment by stephengillie** [comment]
Reset PR

**Pull request closed without merging**